### PR TITLE
Start using Tuples in IDE code.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Tagging @dotnet/roslyn-ide  Currently this doesn't build due to an issue with the VB compiler.

**Comment by CyrusNajmabadi** [comment]
test this please

File: src/Workspaces/Core/Portable/Rename/ConflictEngine/RenamedSpansTracker.cs [hunk 1]
```diff
@@ -180,7 +180,7 @@ internal void ClearDocuments(IEnumerable<DocumentId> conflictLocationDocumentIds
 
         internal bool ContainsResolvedNonReferenceLocation(DocumentId documentId, TextSpan originalLocation)
         {
-            return // (this.documentToModifiedSpansMap.Contains(documentId) && this.documentToModifiedSpansMap[documentId].Contains(t => t.Item1 == originalLocation.SourceSpan)) ||
+            return // (this.documentToModifiedSpansMap.Contains(documentId) && this.documentToModifiedSpansMap[documentId].Contains(t => t.oldSpan == originalLocation.SourceSpan)) ||
```
Comment (jasonmalinowski) [inline src/Workspaces/Core/Portable/Rename/ConflictEngine/RenamedSpansTracker.cs]: Well done!

**Comment by CyrusNajmabadi** [comment]
retest this please

**Pull request merged**